src/Swashbuckle.AspNetCore.ReDoc/Swashbuckle.AspNetCore.ReDoc.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>Middleware to expose an embedded version of ReDoc from an ASP.NET Core application</Description>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
```

Comment:
I'm planning to make netstandard2.1 the new minimum so could strip out 2.0 here and in other places

Reply:
@domaindrivendev I've moved everything over to `netstandard2.1`, but can't find a way to "walk back" the multi-targetting. I haven't been able to produce a build without it that doesn't crash in production with `MissingMethodException`, for the reasons stated in the PR description. Oddly this only occurs when referencing the NuGet packages and not in the test web sites that all target `netcoreapp3.0`.